### ISSUE TITLE
Get only the last result in a temporal filter

### ISSUE THREAD

**Issue by asindl** [CrunchyData/pg_tileserv#190]
Is it possible to sort and limit the number of responses for a certain features set using CQL?
In my example, I have several entries at a set of locations, but at different timestamps. 
With CQL, I can nicely filter for certain timespans. But based on the temporal filter, I might get multiple responses for some locations.
So what I am looking for is the last entry for a location within a given temporal filter. Such as 
```
SORT BY timestamp DESC LIMIT 1
``` 
in SQL...

**Comment by pramsey** [comment]
Yeah, tile format doesn't really have a use for an "order by" so it hasn't been done. That just leaves you the option of writing a function that returns tiles the way you want. Which isn't a bad option, since then you have the full power of SQL available.

**Comment by asindl** [comment]
Thanks for the answer. 
I guess, you are referring to[this] (https://access.crunchydata.com/documentation/pg_tileserv/1.0.3/usage/function-layers/).
I have never used functions as layers. Is there anything else, I need to do, besides creating the function in postgres?
Will it be auto-discovered?

**Comment by pramsey** [comment]
The database is re-scanned every time you hit index.html or index.json. If it has the right signature, it'll be picked up.

**Comment by dr-jts** [comment]
`pg_featureserv` has the `sortby` and `limit` parameters (see [here](https://github.com/CrunchyData/pg_featureserv/blob/master/API.md#parameters)).  These are standard parameters in the OGC API.  Perhaps these could be added to `pg_tileserv`?

**Comment by asindl** [comment]
> `pg_featureserv` has the `sortby` and `limit` parameters (see [here](https://github.com/CrunchyData/pg_featureserv/blob/master/API.md#parameters)). These are standard parameters in the OGC API. Perhaps these could be added to `pg_tileserv`?

I think, my request is a bit more complex, as it needs a DISTINCT as well. 
So the function layer worked perfectly well for me and gives a lot of flexibility.
The only thing I struggled for some hours was the problem with the built-in UI and the visibility of function layers as discussed  [here](https://github.com/CrunchyData/pg_tileserv/issues/134).
It is said [here](https://github.com/CrunchyData/pg_tileserv/commit/34ffdb87797d5feaf3884482ed9c6e15afafa665), that the name has to be set to `default`, to be seen in the UI. But the official [documentation](https://access.crunchydata.com/documentation/pg_tileserv/1.0.3/usage/function-layers/) doesn't state that...
When I named the layer `default` in the `ST_AsMVT` function, I saw the data in the UI as well.
